### PR TITLE
Add depositToken

### PR DESCRIPTION
**Pull request by akaster99**
Add new method depositToken for nft-server.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -647,3 +647,12 @@ export interface WithdrawRequestList {
     };
   };
 }
+export interface DepositTransaction {
```
Comment (platfowner) [inline src/types.ts:650]: minor) could you add an empty line above?

File: src/asset.ts [hunk 2]
```diff
@@ -308,4 +308,14 @@ export default class Asset extends AinftBase {
     const trailingUrl = `credit/${symbol}/lockup`;
     return this.sendRequest(HttpMethod.POST, trailingUrl, body);
   }
+  /**
```
Comment (platfowner) [inline src/asset.ts:311]: indentation? + could you add an empty line above?

File: src/types.ts (hunk 1 above)
Comment (akaster99) [reply src/types.ts:651]: Added, thanks!

File: src/asset.ts (hunk 2 above)
Comment (akaster99) [reply src/asset.ts:312]: Added, thanks!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/asset.ts [hunk 3]
```diff
@@ -308,4 +308,15 @@ export default class Asset extends AinftBase {
...
   }
+  
+  /**
+ * Reflect withdraws complete status to server after transfer tokens
+ * @param {string} appId
+ * @param {DepositTransaction} transaction
+ */
+  depositToken(appId: string, transaction: DepositTransaction): Promise<void> {
+    const body = { appId, transaction };
+    const trailingUrl = `deposit/trnasaction`;
```
Comment (kriii) [inline src/asset.ts:319]: `trnasaction` seems to be a typo.

Comment (akaster99) [reply src/asset.ts]: fixed, thanks!

**Comment by akaster99** [comment]
prettier applied

**Review by yoojinko — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**